src/schema/users.ts

```diff
@@ -154,9 +160,13 @@ export const typeDefs = /* GraphQL */ `
     """
     infoConfirmed: Boolean
```

Comment:
I know that it's not part of this PR and I might have missed that, but this should be provided only to the same logged-in user. For example, if Lee is trying to get Chris's `infoConfirmed` field it should be null for him. Only if Chris tries to get Chris's `infoConfirmed` it should have the right value